### PR TITLE
Chore login alterations

### PR DESCRIPTION
**Pull request by EevanR**
- remedy issue with always asking to login on create new trip
- show who is logged in
- logout (delete local storage session)
- header as component
- current user data to initial state

Screen Shot 2020-02-24 at 9 36 50 PM

### CODE REVIEW

File: src/components/FacebookLogin.jsx [hunk 1]
```diff
@@ -78,7 +82,8 @@ const FacebookLogin = props => {
 
 const mapStateToProps = state => {
   return {
-    progression: state.progression
+    progression: state.progression,
+    current_user: state.current_user
```
Comment (mantanquzzmo) [inline src/components/FacebookLogin.jsx]: camelcase

File: README.md [hunk 2]
```diff
@@ -17,7 +17,13 @@ https://tripwiz.netlify.com/
 ## To run locally
 #### Clone repository
 ```
+$ git clone https://github.com/EevanR/tripwiz_client.git
+```
```
Comment (mantanquzzmo) [inline README.md]: should we just not have craft here?

File: src/components/FacebookLogin.jsx [hunk 3]
```diff
@@ -8,6 +8,10 @@ import { Redirect } from "react-router";
 const FacebookLogin = props => {
   const [redirect, setRedirect] = useState(false);
 
+  if (localStorage.getItem("J-sunkAuth-Storage")) {
```
Comment (mantanquzzmo) [inline src/components/FacebookLogin.jsx:11]: these two can be merged into one

**Review by mantanquzzmo — CHANGES_REQUESTED** [review]

File: src/components/FacebookLogin.jsx (hunk 1 above)
Comment (EevanR) [reply src/components/FacebookLogin.jsx]: changed

File: README.md (hunk 2 above)
Comment (EevanR) [reply README.md]: yeah guess we can

File: src/components/FacebookLogin.jsx (hunk 3 above)
Comment (EevanR) [reply src/components/FacebookLogin.jsx:11]: DONE

File: README.md (hunk 2 above)
Comment (EevanR) [reply README.md]: yeah guess we can

Comment (EevanR) [reply README.md]: YEA DONE

**Review by mantanquzzmo — APPROVED** [review]

**Pull request merged**